ts/components/staking/wizard/market_maker.tsx

```diff
@@ -5,6 +5,7 @@ import { Icon } from 'ts/components/icon';
 import { InfoTooltip } from 'ts/components/ui/info_tooltip';
 
 import { colors } from 'ts/style/colors';
+import { formatEther, formatPercent, formatZrx } from 'ts/utils/format_number';
```

Comment:
? I think some extra commits might have been added

Reply:
merged in `development-staging` again should be fine now